install/config/targets/SYSTEM/cmd_tlm/meta_cmd_tlm.txt

```diff
@@ -0,0 +1,8 @@
+TELEMETRY SYSTEM META BIG_ENDIAN "System Meta Data Telemetry Packet"
```

Comment:
The files are not required, and the packet will be automatically generated without the file if it doesn't exist.  They do exist so that you can modify them and add your project specific meta data.